pkgs/dartpad_ui/lib/utils.dart

```diff
@@ -36,29 +36,12 @@ RelativeRect calculatePopupMenuPosition(
   );
 }
 
-bool hasFlutterWebMarker(String javaScript, {required bool isNewDDC}) {
-  const marker1 = 'window.flutterConfiguration';
-  if (javaScript.contains(marker1)) {
-    return true;
-  }
-  if (isNewDDC &&
-      javaScript.contains('defineLibrary(') &&
-      javaScript.contains('importLibrary("package:flutter/')) {
-    return true;
-    // define('dartpad_main', ['dart_sdk', 'flutter_web']
-  } else if (!isNewDDC &&
-      javaScript.contains("define('") &&
-      javaScript.contains("'flutter_web'")) {
-    return true;
-  }
-
-  return false;
+bool hasFlutterImports(List<String> imports) {
+  return imports.any((import) => import.startsWith('package:flutter/'));
```

Comment:
We can expand this list if we want (though I'm not sure what else would be on it).